eth_utils/currency.py

```diff
@@ -3,42 +3,37 @@
 
 from typing import Union
 
-from .types import (
-    is_integer,
-    is_string,
-)
+from .types import is_integer, is_string
 
 
 units = {
-    'wei':          decimal.Decimal('1'),  # noqa: E241
-    'kwei':         decimal.Decimal('1000'),  # noqa: E241
-    'babbage':      decimal.Decimal('1000'),  # noqa: E241
-    'femtoether':   decimal.Decimal('1000'),  # noqa: E241
-    'mwei':         decimal.Decimal('1000000'),  # noqa: E241
-    'lovelace':     decimal.Decimal('1000000'),  # noqa: E241
-    'picoether':    decimal.Decimal('1000000'),  # noqa: E241
-    'gwei':         decimal.Decimal('1000000000'),  # noqa: E241
-    'shannon':      decimal.Decimal('1000000000'),  # noqa: E241
-    'nanoether':    decimal.Decimal('1000000000'),  # noqa: E241
-    'nano':         decimal.Decimal('1000000000'),  # noqa: E241
-    'szabo':        decimal.Decimal('1000000000000'),  # noqa: E241
-    'microether':   decimal.Decimal('1000000000000'),  # noqa: E241
-    'micro':        decimal.Decimal('1000000000000'),  # noqa: E241
-    'finney':       decimal.Decimal('1000000000000000'),  # noqa: E241
-    'milliether':   decimal.Decimal('1000000000000000'),  # noqa: E241
-    'milli':        decimal.Decimal('1000000000000000'),  # noqa: E241
-    'ether':        decimal.Decimal('1000000000000000000'),  # noqa: E241
-    'kether':       decimal.Decimal('1000000000000000000000'),  # noqa: E241
-    'grand':        decimal.Decimal('1000000000000000000000'),  # noqa: E241
-    'mether':       decimal.Decimal('1000000000000000000000000'),  # noqa: E241
-    'gether':       decimal.Decimal('1000000000000000000000000000'),  # noqa: E241
-    'tether':       decimal.Decimal('1000000000000000000000000000000'),  # noqa: E241
+    "wei": decimal.Decimal("1"),  # noqa: E241
```

Comment:
is it possible to tell black not to touch this block?

Reply:
The best we can do is tell black to ignore this entire file, which seems not so great to me. I agree this isn't super readable, but I'd be more in favor of something like separating the different denominations by groups

```python
units = {
    # 10 ^ 1
    "wei":  decimal.Decimal("1"),  # noqa: E241
    # 10 ^ 3
    "kwei": decimal.Decimal("1000"),  # noqa: E241
    "babbage": decimal.Decimal("1000"),  # noqa: E241
    "femtoether": decimal.Decimal("1000"),  # noqa: E241
    # 10 ^ 6
   ...
```

Reply:
How about we move this to it's own module and tell black to ignore it.  Being able to see how the numbers line up is somewhat required for validating whether these values are correct.